pyqtorch/utils.py

```diff
@@ -32,6 +33,21 @@
 logger = getLogger(__name__)
 
 
+def qubit_support_as_tuple(support: int | tuple[int, ...]) -> tuple[int, ...]:
```

Comment:
Yeah, not sure it is entirely necessary to have a standalone function for this but that is a mere stylistic point.

Reply:
I put it there as it's a frequent operation used everywhere.